scripts/update-dependencies/NuGet.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```

Comment:
Why do we need this nuget.config file? Can't we use: https://github.com/dotnet/corefxlab/blob/master/NuGet.Config#L15